### DISCUSSION TITLE
Merge question. Noob

### DISCUSSION THREAD

**Discussion by themortenbay** [DivinumOfficium/divinum-officium#2988] [Q&A] [answered]
Oh no, oh no... I just realised I've been overwriting the files in the Horas/English directory for my Danish translation (in my fork) - instead of creating new documents in my Horas/Dansk folder.. If I move all the changed files into the correct folder, would it be easy enough for one of you to only merge my new folder - and just leave the Horas/English folder up-to-date? Or will I, like I fear, be forced to create a new fork and manully copy all the documents over there?

**Comment by mbab** [comment, marked as answer]
Commiting changes only in Dansk folder (not English) should sufice (or You can simply checkout files in English to restore them).
